deploy/runtools/firesim_topology_elements.py

```diff
@@ -333,13 +333,37 @@ def copy_back_job_results_from_run(self, slotno: int) -> None:
 
         dest_sim_dir = self.get_host_instance().get_sim_dir()
 
+        def has_sudo() -> bool:
+            with warn_only():
+                return run("sudo -ln true").return_code == 0
+
+        def mount(img: str, mnt: str, tmp_dir: str) -> None:
+            if has_sudo():
+                run(f"sudo mount -o loop {img} {mnt}")
+            else:
+                run(f"""screen -S guestmount-wait -dm bash -c "guestmount --pid-file {tmp_dir}/guestmount.pid -a {img} -m /dev/sda {mnt}; while true; do sleep 1; done;" """, pty=False)
+                run(f"""while [ ! "$(ls -A {mnt})" ]; do echo "Waiting for mount to finish"; sleep 1; done""")
```

Comment:
Probably should add a timeout here so that if the mount somehow fails, it errors out nicely instead of spinning forever.